x-pack/plugins/maps/common/constants.ts

```diff
@@ -25,7 +25,6 @@ export const EMS_TILES_VECTOR_TILE_PATH = 'vector/tile';
 export const MAP_SAVED_OBJECT_TYPE = 'map';
```

Comment:
I think this constant needs to match the `name` in `x-pack/plugins/maps/server/saved_objects/maps_telemetry.ts`: `maps-telemetry`. Because we are searching for it in the `getMapsTelemetry` method :)

Reply:
Hold on! Scratch that! Is this related to some other saved objects maps is maintaining?

Reply:
Yeah, looks like you figured this out in the above conversation but this is unrelated. This is examining saved maps of type `map`.